app/styles/app/layouts/spotlight/insights-heatmap.scss

```diff
@@ -0,0 +1,103 @@
+.heatmap-container {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  background-color: white;
+  line-height: 1.25;
+}
+.range-selector-heatmap {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+.day-label-container {
+  display: flex;
+  flex-direction: column;
+  margin-left: -10px;
+  margin-right: 7px;
+}
+.day-label {
+  color: black;
+  opacity: 0.8;
+  font-size: 18px;
+  padding: 1.5px;
+}
+.day-label-heatmap-container {
+  display: flex;
+  flex-direction: row;
+  margin-top: 16px;
+}
+.build-filter-label {
+  height: 31px;
+  display: inline-block;
+  text-align: center;
+  vertical-align: middle;
+  cursor: pointer;
+  border: 1px solid #ccc;
+  padding: 6px 12px;
+  line-height: 1.42857143;
+  border-radius: 4px;
+  color: #333;
+  background-color: #fff;
+  outline: none;
+}
+.build-filter-label:hover {
+  background-color: #d4d4d4;
+  border-color: #8c8c8c;
+}
+.caret {
+  width: 0;
+  height: 0;
+  display: inline-block;
+  border: 5px solid transparent;
+  margin-left: 10px;
+  margin-bottom: -2px;
+}
+.caret.down {
+  border-top-color: black;
+}
+.box {
+  width: 15px;
+  height: 15px;
+}
+.failed {
+  background-color: #db4545;
+}
+.all {
+  background-color: #04c2bf;
+}
+.successful {
+  background-color: #32ab52;
+}
+.errored {
+  background-color: #db5c07;
+}
+.canceled {
```

Comment:
@chinmaychaudhari-accolite why you are using generic names for classes?
the UI is broken!!!

Reply:
Heatmap css generic names are removed and changed.

Reply:
![Screenshot from 2022-03-15 12-37-33](https://user-images.githubusercontent.com/97581436/158324717-d2eb8c7e-3bd0-445a-844d-77d53d21d042.png)